src/identity/IdentityPage.test.tsx

```diff
@@ -443,6 +451,14 @@ describe('Identity page', () => {
   });
 });
 
+function getTestResultsLink() {
+  return screen.getByTestId('test-result-link');
```

Comment:
Any reason to not use the text?

Reply:
'results' appears in two places on the page, so it finds two different elements.

Reply:
We should be able to `getByText('Results')`, though  — for a title, exact match querying is more acceptable.